e2e/utils.go

```diff
@@ -498,3 +498,17 @@ func oneReplicaDeployYaml(template string) string {
 	var re = regexp.MustCompile(`(\s+replicas:) \d+`)
 	return re.ReplaceAllString(template, `$1 1`)
 }
+
+// nolint:unparam // currently major is always 1, this can change in the future
+func k8sVersionGreaterEquals(c kubernetes.Interface, major, minor int) bool {
+	v, err := c.Discovery().ServerVersion()
+	if err != nil {
+		e2elog.Failf("failed to get server version with error with error %v", err)
```

Comment:
FailF will mark the test case as a failure. this need to be replaced with a log and return will not happen

Reply:
It is actually intended to fail the case if fetching the ServerVersion errors out. Returning from this function would cause the case to be silently skipped (version not matching), which is not what we want. (OK, the return on the next line is unneeded, I can remove that and leave a comment instead.)

Reply:
@Madhu-1 is the added comment sufficiently clear to explain the working of the new function?

Reply:
Sounds good @nixpanic

Reply:
@nixpanic 
s/with error with error/with error/ ?

Reply:
Oh, yes! This is actually coming from the original version checking code that was moved from other files into its own function. I'll fix it.